<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportAudio</class>
 <widget class="QWidget" name="ExportAudio">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ExportAudio</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>60</y>
     <width>118</width>
     <height>256</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
    <item>
     <widget class="QGroupBox" name="groupFormat">
      <property name="title">
       <string>Select Format</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <widget class="QRadioButton" name="radioWAV">
         <property name="text">
          <string>WAV (PCM)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioRAW">
         <property name="text">
          <string>RAW (PCM)</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupSample">
      <property name="title">
       <string>Sample Datatype</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <widget class="QRadioButton" name="radioSInt8">
         <property name="text">
          <string>Signed Int8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioUInt8">
         <property name="text">
          <string>Unsigned Int8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioSInt16">
         <property name="text">
          <string>Signed Int16</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioUInt16">
         <property name="text">
          <string>Unsigned Int16</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioSInt32">
         <property name="text">
          <string>Signed Int32</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioUInt32">
         <property name="text">
          <string>Unsigned Int32</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioFloat">
         <property name="text">
          <string>Float32</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>60</y>
     <width>171</width>
     <height>172</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Samplerate:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinSampleRate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string> Hz</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>6144000</number>
        </property>
        <property name="singleStep">
         <number>1000</number>
        </property>
        <property name="value">
         <number>48000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Maximum length:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="spinTime">
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="time">
         <time>
          <hour>0</hour>
          <minute>30</minute>
          <second>0</second>
         </time>
        </property>
        <property name="maximumTime">
         <time>
          <hour>23</hour>
          <minute>59</minute>
          <second>59</second>
         </time>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::MinuteSection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelMemory">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Required memory: ??? MB</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupNormalize">
      <property name="title">
       <string>Normalize amplitude</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <widget class="QRadioButton" name="radioNoNormalize">
         <property name="text">
          <string>No normalization</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioNormalizeClipping">
         <property name="text">
          <string>Only if clipping</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioNormalizePeak">
           <property name="text">
            <string>Set peak to:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinPeak">
           <property name="suffix">
            <string> %</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>95</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="buttExport">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>285</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Export!</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIntro">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>296</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This will allow you to generate an audio file from MusicString code &lt;span style=&quot; font-weight:600;&quot;&gt;SOURCENAME&lt;/span&gt;. Please select desired options and click Export. Depending on length, it might take a while.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
